src/main/java/capstone/mfslbackend/service/TransactionService.java

```diff
@@ -55,9 +62,56 @@ public Transaction createTransaction(Long fantasyTeamId, Long incomingPlayerId,
         }
 
         Player playerIn = playerService.getPlayerById(incomingPlayerId);
-        transaction.setPlayerIn(playerIn);
+        Player outgoingPlayer = playerService.getPlayerById(outgoingPlayerId);
+        String position = playerIn.getPosition();
+
```

Comment:
Sorry I didnt mention this before, I just thought of this, but we might want to make this a helper function, called validateTransaction() or something to make sure its a legal transaction

Reply:
because were gonna need to use this stuff again